truffle-config.arbitrum.js

```diff
@@ -43,12 +49,27 @@ module.exports = {
         })
       }
     },
+    arbitrum_goerli: {
+      network_id: 421613,
+      provider: function() {
+        return new HDWalletProvider({
+          mnemonic: {
+            phrase: goerliMnemonic
+          },
+          providerOrUrl: 'https://arbitrum-goerli.infura.io/v3/' + infuraKey,
+          addressIndex: 0,
+          numberOfAddresses: 1,
+          network_id: 421613,
+          chainId: 421613
+        })
+      }
+    },
     arbitrum_testnet: {
```

Comment:
This name is now a bit broad given two testnets listed, maybe we rename it to `arbitrum_rinkeby`?  I'm also open to removing the `arbitrum_` from all the network names since that should be obvious from running a command like `truffle migrate --network=goerli --config=truffle-config.arbitrum`. Am I forgetting situations where this would be less obvious?

Reply:
I agree. If it is within the Arbitrum config I have no problem with removing `arbitrum_` from the network names.

Reply:
On second though, I think I'll leave these out of this PR to follow suit with the other L2 boxes. I'll update them all at once if we decide it's worth it.